docs/topics/gradle/gradle-configure-project.md

```diff
@@ -50,7 +50,8 @@ In the following table, there are the minimum and maximum **fully supported** ve
 
 | KGP version   | Gradle min and max versions            | AGP min and max versions                            |
 |---------------|----------------------------------------|-----------------------------------------------------|
-| 2.1.0         | %minGradleVersion%–%maxGradleVersion%* | %minAndroidGradleVersion%–%maxAndroidGradleVersion% |
+| 2.1.20        | %minGradleVersion%–%maxGradleVersion%* | %minAndroidGradleVersion%–%maxAndroidGradleVersion% |
```

Comment:
Should `*` be at the end? `withJava()` is deprecated in this release and user could have KMP project without deprecations.

Reply:
You're right. Fixed!